packages/server/src/db/index.js

```diff
@@ -585,20 +586,35 @@ function getAgencyKeywords(agencyId) {
         .where('agency_id', agencyId);
 }
 
-async function createAgency({
-    name, abbreviation, parent, warning_threshold, danger_threshold,
-}) {
+async function createAgency(agency) {
     // seeded agencies with hardcoded ids will make autoicrement fail since it doesnt
     // know which is the next id
     await knex.raw('select setval(\'agencies_id_seq\', max(id)) from agencies');
-    return knex(TABLES.agencies)
-        .insert({
+
+    // Enforce tenant isolation by using the tenant_id from the user
+    // and the main_agency_id from the tenant.
```

Comment:
Currently requireAdminUser and requireUser do not enforce tenant isolation in the isAuthorized function. I wanted to leave that for after this PR and the backfill. So this is just step one, so that we can only backfill one more time, then add tenant isolation to the endpoints, which is where I would update isAuthorized to work at the tenant level.

Reply:
It does actually enforce for routes nested under `:organizationId` paths (which is almost all; see [here](https://github.com/usdigitalresponse/usdr-gost/blob/10f8b3eb319fce091397c5eb1a646ed246b61c07/packages/server/src/configure.js#L30-L41)) that the user is a member of the specified agency (or in case of admins, can be of a superagency). This implicitly also ensures the specified agency is in the right tenant too (you could add an assert I guess that user's and agency's tenant match; that should be an invariant). Is there some other type of check I'm not thinking of that you're thinking of adding to `requireUser`?

Most APIs that deal with other types of entities already have checks in application code that the entity is associated with the specified agency -- though for those (i.e. listed in #72) that are supposed to become tenant-scoped rather than agency-scoped, there will need to be some changes.

As I [mentioned](https://usdigitalresponse.slack.com/archives/C0324KDQSCR/p1660685357263199), there's at least one place that specifies an agency ID separately from `:organizationId` though; we should change that.